src/CSSTransitionGroupChild.js

```diff
@@ -19,10 +19,12 @@ import { addEndEventListener, removeEndEventListener } from './TransitionEvents'
 const TICK = 17;
 
 export class CSSTransitionGroupChild extends Component {
-	transition(animationType, finishCallback) {
-		let node = getComponentBase(this),
-			className = this.props.name[animationType] || this.props.name + '-' + animationType,
-			activeClassName = this.props.name[animationType + 'Active'] || className + '-active';
+	transition(animationType, finishCallback, timeout) {
+		let node = getComponentBase(this);
+
+		let className = this.props.name[animationType] || this.props.name + '-' + animationType;
```

Comment:
Hi @hassanbazzi & @developit : 
I didn't do anything particular here. I've just removed the group coma declaration. Nothing else !
And I think `className` is still needed 👉  https://github.com/myarcane/preact-css-transition-group/blob/663cd4e90dc8476290f955b5829d60ced9d89ff3/src/CSSTransitionGroupChild.js#L56